tests/integration/long/test_ssl.py

```diff
@@ -193,6 +242,62 @@ def test_can_connect_with_ssl_client_auth(self):
 
         cluster.shutdown()
 
+    def test_can_connect_with_ssl_client_auth_host_name(self):
+        """
+        Test to validate that we can connect to a C* cluster that has client_auth enabled, and hostmatching
+
+        This test will setup and use a c* cluster that has client authentication enabled. It will then attempt
+        to connect using valid client keys, and certs (that are in the server's truststore), and attempt to preform some
+        basic operations, with match_hostname specified
+        @jira_ticket PYTHON-296
+        @since 3.3
+
+        @expected_result The client can connect via SSL and preform some basic operations
+
+        @test_category connection:ssl
+        """
+
+        # Need to get absolute paths for certs/key
```

Comment:
These tests seem almost identical aside from additional ssl options. Is it worth refactoring?